pybossa.js

```diff
@@ -17,36 +17,43 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 (function(pybossa, $, undefined) {
-    var url = '/';
-    var _userId;
-    var _observer;
+    let URL = '/';
```

Comment:
For PR simplicity, should we consider reverting the change in casing and type for these variables? This would revert a significant number of lines from change.